src/game/common/rts/playerlist.cpp

```diff
@@ -18,3 +18,234 @@
 #else
 PlayerList *g_thePlayerList = nullptr;
 #endif
+
+// zh: 0x0045A7C0 wb: 0x007BEE10
+PlayerList::PlayerList() : m_local(nullptr), m_playerCount(0)
+{
+    for (auto i = 0; i < MAX_PLAYER_COUNT; ++i) {
+        m_players[i] = new Player(i);
+    }
+    Init();
+}
+
+// zh: 0x0045A8D0 wb: 0x007BEEFD
+PlayerList::~PlayerList()
+{
+    Init();
```

Comment:
Add a todo to investigate as this is quite illogical

Reply:
Init clears out a lot of things and unloads the PlayerTemplate I think that is the reason for it. Init seems to get called a lot.